File: samples/python/writer.py
from pathlib import Path
from datetime import datetime

FOLDER = Path(r"C:\Fatura")

def write_csv(kind, invoice_no, content):
    FOLDER.mkdir(parents=True, exist_ok=True)
    timestamp = datetime.now().strftime("%Y%m%d%H%M%S%f")[:17]
    final_path = FOLDER / f"{kind}_{invoice_no}_{timestamp}.csv"
    tmp_path = Path(str(final_path) + ".tmp")
    tmp_path.write_text(content, encoding="utf-8", newline="")
    tmp_path.replace(final_path)
    return final_path

File: samples/GO/writer.go
package main
import("fmt";"os";"path/filepath";"time")
func writeCSV(kind,no,content string)(string,error){d:=`C:\Fatura`;if err:=os.MkdirAll(d,0755);err!=nil{return "",err};ts:=time.Now().Format("20060102150405")+fmt.Sprintf("%03d",time.Now().Nanosecond()/1e6);f:=filepath.Join(d,fmt.Sprintf("%s_%s_%s.csv",kind,no,ts));t:=f+".tmp";if err:=os.WriteFile(t,[]byte(content),0644);err!=nil{return "",err};if err:=os.Rename(t,f);err!=nil{return "",err};return f,nil}
